source/index.html.erb

```diff
@@ -2,24 +2,27 @@
 title: Ralph's Launch Cards
 ---
 
-<nav class="card-categories">
-
-</nav>
+  <details>
+  <summary>Topics</summary>
+  <ul class="">
+    <li><a class="topic__link" href="#">Productivity</a></li>
+    <li><a class="topic__link" href="#">Management</a></li>
+    <li><a class="topic__link" href="#">Food</a></li>
+  </ul>
+</details>
 
 <div class="card">
-  <header class="card__category">
+  <header id="card-topic-header" class="card__category">
     Management
```

Comment:
Should we have some placeholder text in the topic and prompt tags when the page loads, instead of this?